src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1213,6 +1293,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
             sectionsList.forEach(section => {
                 if (section.isCreditCardWidgetAvailable) {
                     section.setCardHolderName(this.fabricateCardHolderName(fieldList));
+                    GeFormService.setFabricatedCardholderNames(this.fabricateCardHolderName(fieldList));
```

Comment:
If those other methods from GeFormService are moved into the App, then this call might need to be reworked also as it might not make sense to set the names on the Service component any longer after that.

Reply:
Updated this to set a public property in `geFormRenderer` that `geGiftEntryApp` can retrieve.